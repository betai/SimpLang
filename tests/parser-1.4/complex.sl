    if (1<2) then (3*4) else (5+!-if 7 then 8 else 9 end) end
