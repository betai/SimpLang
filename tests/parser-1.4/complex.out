if
  <
    1
    2
  *
    3
    4
  +
    5
    !
      -
        if
          7
          8
          9
